src/manifolds/Grassmann.jl

```diff
@@ -51,223 +49,235 @@ The manifold is named after
         )
     end
-    if size(x) != representation_size(M)
+    if size(p) != representation_size(M)
         return DomainError(
-            size(x),
-            "The matrix $(x) is does not lie on the Grassmann manifold of dimension ($(n),$(k)), since its dimensions are wrong.",
+            size(p),
+            "The matrix $(p) is does not lie on the Grassmann manifold of dimension ($(n),$(k)), since its dimensions are wrong.",
         )
     end
-    c = x' * x
+    c = p' * p
     if !isapprox(c, one(c); kwargs...)
         return DomainError(
             norm(c - one(c)),
-            "The point $(x) does not lie on the Grassmann manifold of dimension ($(n),$(k)), because x'x is not the unit matrix.",
+            "The point $(p) does not lie on the Grassmann manifold of dimension ($(n),$(k)), because x'x is not the unit matrix.",
         )
     end
 end
 
-@doc doc"""
-    check_tangent_vector(M::Grassmann{n,k,F}, x, v)
+@doc raw"""
+    check_tangent_vector(M::Grassmann{n,k,F}, p, X)
 
-Check whether `v` is a tangent vector in the tangent space of `x` on the [`Grassmann`](@ref)
-`M`, i.e. that `v` is of size and type as well as that
+Check whether `X` is a tangent vector in the tangent space of `p` on
+the [`Grassmann`](@ref) `M`, i.e. that `X` is of size and type as well as that
 
 ````math
-    x^{\mathrm{H}}v + v^{\mathrm{H}}x = 0_k,
+    p^{\mathrm{H}}X + X^{\mathrm{H}}p = 0_k,
 ````
 
 where $\cdot^{\mathrm{H}}$ denotes the complex conjugate transpose or Hermitian and $0_k$
-denotes the $k\times k$ zero natrix.
+denotes the $k \times k$ zero natrix.
 """
-function check_tangent_vector(G::Grassmann{n,k,F}, x, v; kwargs...) where {n,k,F}
-    t = check_manifold_point(G, x)
+function check_tangent_vector(G::Grassmann{n,k,F}, p, X; kwargs...) where {n,k,F}
+    t = check_manifold_point(G, p)
     t === nothing || return t
-    if (F === ℝ) && !(eltype(v) <: Real)
+    if (F === ℝ) && !(eltype(X) <: Real)
         return DomainError(
-            eltype(v),
-            "The matrix $(v) is not a real-valued matrix, so it can not be a tangent vector to the Grassmann manifold of dimension ($(n),$(k)).",
+            eltype(X),
+            "The matrix $(X) is not a real-valued matrix, so it can not be a tangent vector to the Grassmann manifold of dimension ($(n),$(k)).",
         )
     end
-    if (F === ℂ) && !(eltype(v) <: Real) && !(eltype(v) <: Complex)
+    if (F === ℂ) && !(eltype(X) <: Real) && !(eltype(X) <: Complex)
         return DomainError(
-            eltype(v),
-            "The matrix $(v) is neiter real- nor complex-valued matrix, so it can not bea tangent vector to the complex Grassmann manifold of dimension ($(n),$(k)).",
+            eltype(X),
+            "The matrix $(X) is neiter real- nor complex-valued matrix, so it can not bea tangent vector to the complex Grassmann manifold of dimension ($(n),$(k)).",
```

Comment:
```suggestion
            "The matrix $(X) is neither a real- nor complex-valued matrix, so it can not be a tangent vector to the complex Grassmann manifold of dimension ($(n),$(k)).",
```